src/detect-engine-content-inspection.c

```diff
@@ -360,23 +361,21 @@ uint8_t DetectEngineContentInspection(DetectEngineCtx *de_ctx, DetectEngineThrea
                         /* see if the next buffer keywords match. If not, we will
                          * search for another occurrence of this content and see
                          * if the others match then until we run out of matches */
-                        uint8_t r = DetectEngineContentInspection(de_ctx, det_ctx, s, smd + 1, p, f,
-                                buffer, buffer_len, stream_start_offset, flags, inspection_mode);
+                        int r = DetectEngineContentInspectionInternal(de_ctx, det_ctx, s, smd + 1,
```

Comment:
Don't you want to use rather those specific int (e.g. int32_t) types?

Reply:
for simple case like this I don't think it matters very much

Reply:
I thought we were trying to slowly replace it in all places possible.